examples/fuse_examples/imaging/classification/stoic21/runner_stoic21.py

```diff
@@ -207,51 +205,54 @@ def run_train(paths: dict, train_params: dict):
     # ====================================================================================
     # Metrics
     # ====================================================================================
-    metrics = OrderedDict([
+    train_metrics = OrderedDict([
         ('auc', MetricAUCROC(pred='model.output.classification', target="data.gt.probSevere")),
     ])
 
-    # =====================================================================================
-    #  Callbacks
-    # =====================================================================================
-    callbacks = [
-        # default callbacks
-        TensorboardCallback(model_dir=paths['model_dir']),  # save statistics for tensorboard
-        MetricStatisticsCallback(output_path=paths['model_dir'] + "/metrics.csv"),  # save statistics a csv file
-        TimeStatisticsCallback(num_epochs=train_params['manager.train_params']['num_epochs'], load_expected_part=0.1)  # time profiler
-    ]
+    validation_metrics = copy.deepcopy(train_metrics) # use the same metrics in validation as well
+ 
+    # either a dict with arguments to pass to ModelCheckpoint or list dicts for multiple ModelCheckpoint callbacks (to monitor and save checkpoints for more then one metric). 
+    best_epoch_source = dict(
+        monitor="validation.metrics.auc",
+        mode="max",
+    )
+    
+    # create optimizer
+    optimizer = optim.SGD(model.parameters(), lr=train_common_params['opt.lr'], weight_decay=train_common_params['opt.weight_decay'], momentum=0.9, nesterov=True)
```

Comment:
Just a minor comment regarding the documentation:
We split the code in `run_train` into sections using (for example):
```
# ==========
# Model
# ==========
```

Now I noticed that the optimizer and lr_sched are assigned to `#  Metrics` section. 
Do you think we should add another one section between `# Metrics` and `# Train` ? (Also for the other examples and the walkthrough_template)

Reply:
Fixed it, thanks